src/main/java/com/epam/ta/reportportal/util/ReportingQueueService.java

```diff
@@ -40,10 +42,9 @@
 	 * @return
 	 */
 	public String getReportingQueueKey(String launchUuid) {
-		int value = UUID.fromString(launchUuid).hashCode();
+		int value = launchUuid.matches(UUID_REGEX) ? UUID.fromString(launchUuid).hashCode() : launchUuid.hashCode();
```

Comment:
what about updating existing unit test or creating one if not exists?

Reply:
done